src/govuk/components/character-count/character-count.mjs

```diff
@@ -253,51 +297,25 @@ CharacterCount.prototype.getCountMessage = function () {
  * If there is no configured threshold, it is set to 0 and this function will
  * always return true.
  *
- * @returns {Boolean} true if the current count is over the options.threshold
+ * @returns {Boolean} true if the current count is over the config.threshold
  *   (or no threshold is set)
  */
 CharacterCount.prototype.isOverThreshold = function () {
+  // No threshold means we're always above threshold
+  // so we can save some computation
```

Comment:
I like this optimisation – makes sense 👍🏻 

Suggest wrapping at 80 characters and rewording slightly to fit on one line

```suggestion
  // No threshold means we're always above threshold so save some computation
```